src/StackHCI/Az.StackHCI.psm1

```diff
@@ -2980,15 +3049,1421 @@ param(
         Write-Error -Exception $_.Exception -Category OperationStopped -ErrorAction Continue
         # Get script line number, offset and Command that resulted in exception. Write-Error with the exception above does not write this info.
         $positionMessage = $_.InvocationInfo.PositionMessage
-        Write-Error ("Exception occured in Test-AzStackHCIConnection : " + $positionMessage) -Category OperationStopped
+        Write-Error ("Exception occurred in Test-AzStackHCIConnection : " + $positionMessage) -Category OperationStopped
+        throw
+    }
+    finally
+    {
+        Stop-Transcript | out-null
+    }
+}
+
+<#
+    .Description
+    Set-AzStackHCI modifies resource properties of the Microsoft.AzureStackHCI cloud resource representing the on-premises cluster to enable or disable features.
+    .PARAMETER ComputerName
+    Specifies one of the cluster node in on-premise cluster that is being registered to Azure.
```

Comment:
Specifies one of the cluster nodes in on-premise cluster that is registered to Azure ( without "being")